### PR TITLE
README: Example on getting the tokens you should remove

### PR DESCRIPTION
**Pull request by johanforssell**
Example of how to find the tokens which you couldn't send to.
I'm maching the two arrays together and then filtering it to get a list of tokens.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,14 @@ sender.send(message, { registrationTokens: registrationTokens }, 10, function (e
   if(err) console.error(err);
   else    console.log(response);
 });
+
+// Collect device registration tokens that you couldn't send messages to
+sender.send(message, { registrationTokens: registrationTokens }, function (err, response) {  
+  var failed_tokens = response.results
+    .map((res, i) => !!res.error ? registrationTokens[i] : undefined)
+    .filter(token => !!token);
```
Comment (hypesystem) [inline README.md]: Thank you for the time you took to contribute :smile:

Here are my thoughts:

Isn't the code wrong, mapping all responses without errors (where `res.error` is falsy, so `!!res.error == false`) to `undefined`? Shouldn't it be the opposite?

I would prefer using `null` to `undefined` in the code example, too.

In addition, I think the `!!` are adding confusion, as the evaluation would be the same without them...

Other than that, it is a good example :smile:

**Comment by johanforssell** [comment]
My thinking is that the `.map()` goes through the array of results. Where there is an error, pick the token from the other array. Otherwise, pick nothing/null. Remember, we're collecting the tokens of erronous sends.

**Comment by hypesystem** [comment]
Ah yes, you are absolutely right, I was just reading it wrong! I blame being tired :smile: 

Maybe we could avoid people like me misunderstanding the code in the future by adding a bit more of a comment?

**Comment by hypesystem** [comment]
This looks good to me! Remember to add yourself to contributors in [package.json](https://github.com/ToothlessGear/node-gcm/blob/master/package.json) :smile:

@eladnava any comments on this before we merge?

**Comment by eladnava** [comment]
Just reviewed, looks good to me! 💯 Thanks so much @johanforssell for the excellent work.

**Comment by hypesystem** [comment]
Thanks for the contribution @johanforssell ! :smile:

**Pull request merged**